test/test_respondent_controller.py

```diff
@@ -1970,3 +1971,29 @@ def test_respondent_emails_are_case_insensitive(self):
         mock_respondent = self.mock_respondent.copy()
         mock_respondent["emailAddress"] = "A@z.com"
         self.post_to_respondents(payload=mock_respondent, expected_status=409)
+
+    def test_get_enrolments_by_survey_business_id(self):
```

Comment:
These tests go against the structure of the others, as I wanted to avoid using endpoints to populate the data to test controllers, we really shouldn't be doing that and split the app and endpoints away from the controller tests, it makes the tests cumbersome, difficult to maintain and slow.  It does however look a bit odd mixing them